content-scripts/inject/module.js

```diff
@@ -302,7 +302,7 @@ if (isScratchGui || isProject) {
   }
 }
 
-if (location.pathname === "/discuss/3/topic/add/") {
+if (location.pathname.match(/^(?:https?:\/\/scratch\.mit\.edu\/)?(topic|\/discuss\/settings)?/)) {
```

Comment:
The `editingScreens` match uses this:
```suggestion
// Same as editingScreens match
if (location.pathname.match(/^\/discuss\/(?:topic\/\d+|\d+\/topic\/add|post\/\d+\/edit|settings\/[\w-]+)\/?$/) {
```
But now that I think about it it's fine if the match isn't perfect.